app/src/main/java/com/topjohnwu/magisk/ui/base/BaseUIActivity.kt

```diff
@@ -96,6 +97,10 @@ abstract class BaseUIActivity<ViewModel : BaseViewModel, Binding : ViewDataBindi
         delegate.onResume()
     }
 
+    override fun dispatchKeyEvent(event: KeyEvent): Boolean {
+        return currentFragment?.onKeyEvent(event) as Boolean || super.dispatchKeyEvent(event)
```

Comment:
This will crash the application when `currentFragment == null`. I suggest changing the the forced cast to optional like so:
```kotlin
return currentFragment?.onKeyEvent(event) == true || super.dispatchKeyEvent(event)
```